src/install/snowflake/configure-integration.mdx

```diff
@@ -0,0 +1,38 @@
+---
+componentType: default
+headingText: Configure the integration
+---
+
+1. Run this command in your CLI to clone the New Relic Snowflake repo:
+
+  ```bash
+  git clone https://github.com/newrelic/newrelic-snowflake-integration.git
+  ```
+
+2. From the `releases` section, download the [`snowflakeintegration-linux`
+binary](https://github.com/newrelic/newrelic-snowflake-integration/releases), and place it in the cloned repo. Make sure the binary is executable by running:
+
+  ```bash
+  chmod +x snowflakeintegration-linux
+  ```
+
+3. To ensure you're using the newest info for the integration, update your `config.yaml` file. Make the file match the snippet in Snowflake's [Key Pair authentication](https://github.com/newrelic/newrelic-snowflake-integration#key-pair-authentication) section. 
+
+4. Run this command to know if you're on a system using a `systemd` service:
```

Comment:
I'd combine steps 4 and 5. Something like:

"If the agent is running as a systemd file [info here]. Otherwise.... [other info here]"

Reply:
Good idea!